api/client/canary/s3.pb.go

```diff
@@ -27,15 +27,15 @@ const (
 const _ = proto.ProtoPackageIsVersion4
 
 // Configuration for the S3 canary integration. If enabled, you must
-// also configure at least one canary.AwsAccount with a list of
-// supportedTypes that includes canary.SupportedType.CONFIGURATION_STORE and/or
-// canary.SupportedType.OBJECT_STORE.
+// also configure at least one `canary.AwsAccount` with a list of
+// `supportedTypes` that includes `canary.SupportedType.CONFIGURATION_STORE`
+// and/or `canary.SupportedType.OBJECT_STORE`.
 type S3 struct {
 	state         protoimpl.MessageState
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// Whether the integration is enabled.
+	// Whether the S3 canary integration is enabled.
```

Comment:
Not sure how to re-word this, because this has to be used with `kayenta.aws.accounts` 
And its to enable a storage engine more than it is a `canary intagration`